tests/test_tuya_spells.py

```diff
@@ -186,39 +135,3 @@ def test_tuya_spell_devices_valid():
                 f"Enable at least one Tuya spell by setting `TUYA_SPELL_READ_ATTRIBUTES` or `TUYA_SPELL_DATA_QUERY` "
                 f"or inherit CustomDevice rather than EnchantedDevice."
             )
-
-        enchanted_clusters = 0  # number of clusters subclassing TuyaEnchantableCluster
-        tuya_cluster_exists = False  # cluster subclassing TuyaNewManufCluster existing
-
-        # iterate over all clusters in the replacement
-        for endpoint_id, endpoint in quirk.replacement[ENDPOINTS].items():
-            if endpoint_id != 1:  # spell is only activated on endpoint 1 for now
-                continue
-            for cluster in endpoint[INPUT_CLUSTERS] + endpoint[OUTPUT_CLUSTERS]:
-                if not isinstance(cluster, int):
-                    # count all clusters which would apply the spell on bind()
-                    if issubclass(cluster, TuyaEnchantableCluster):
-                        enchanted_clusters += 1
-                    # check if there's a valid Tuya cluster where the id wasn't modified
-                    if (
-                        issubclass(cluster, TuyaNewManufCluster)
-                        and cluster.cluster_id == TuyaNewManufCluster.cluster_id
-                    ):
-                        tuya_cluster_exists = True
-
-        # an EnchantedDevice must have exactly one enchanted cluster on endpoint 1
-        if enchanted_clusters == 0:
-            pytest.fail(
-                f"{quirk} does not have a cluster subclassing `TuyaEnchantableCluster` on endpoint 1 "
-                f"as required by the Tuya spell."
-            )
-        elif enchanted_clusters > 1:
-            pytest.fail(
-                f"{quirk} has more than one cluster subclassing `TuyaEnchantableCluster` on endpoint 1"
-            )
-
-        # an EnchantedDevice with the data query spell must also have a cluster subclassing TuyaNewManufCluster
-        if quirk.tuya_spell_data_query and not tuya_cluster_exists:
-            pytest.fail(
-                f"{quirk} set Tuya data query spell but has no cluster subclassing `TuyaNewManufCluster` on endpoint 1"
-            )
```

Comment:
Just noticed that this small part is still required, so I'll add that back.
That's the "second spell" for which the command is on the `TuyaNewManufCluster`.

Reply:
https://github.com/zigpy/zha-device-handlers/pull/3414/commits/23f47f07b90a485aa85e3674c842b30550d8310a